### DISCUSSION TITLE
为什么不准用中文 , 我偏用 , 谁看不懂谁尴尬 !

### DISCUSSION THREAD

**Discussion by hqb72** [community/community#30620] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/30142521/186346991-f1974572-3898-4e83-82d6-fa0f8916eef7.png)

**Comment by davecheney** [comment]
@hqb72 we do not prevent Chinese users. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by Gtd232** [reply]
我想他的意思是,有人发了一个标题为中文的discussions,另一个人说:"你在这种项目下讨论麻烦用英文",他认为不应该限制语言.  
(I think what he meant was that someone posted a discussion with a Chinese title and another person said "you have trouble discussing in English under this kind of project" and he thought there should be no language restrictions)

**Reply by hqb72** [reply]
是的嘞!

**Reply by davecheney** [reply]
There are certainly no restrictions on what language to use on this forum.
